DiSCOS/DiSCOS.md

```
@@ -42,7 +42,7 @@ Non-invasive Altruistic Digital Twin of Duran Serkan KILIÇ(Author) for Collabor
 - Attention to DiSCOS
 - Emotional Intelligence
 - Executive Presence=Decorum * Competency * Integrity * Situational Awareness
-- Reliability
+- Reliability=Secure * Observable * Maintainable * Performant * Scalable
```

markdownlint-cli2 (0.21.0)

[warning] Spaces inside emphasis markers

(MD037, no-space-in-emphasis)